deps/spidershim/test/value.cc

```diff
@@ -32,8 +32,22 @@ static int StrNCmp16(uint16_t* a, uint16_t* b, int n) {
   }
 }
 
+// Based on the function of the same name in test-api.cc, but modified
+// to compare strings in a SpiderShim-compatible way.
+static bool SameSymbol(Local<String> s1, Local<String> s2) {
+  Isolate* isolate = Isolate::GetCurrent();
+  // Use Persistent<T> to get access to the protected Value "equal to" operator.
```

Comment:
(Of course it's also possible that my assumptions about Value and equivalence are wrong, even though the tests pass. If so, please enlighten me!)